engine/src/io/data_provider/UriDataProvider.cpp

```diff
@@ -17,16 +17,17 @@
 namespace ral {
 namespace io {
 
-uri_data_provider::uri_data_provider(std::vector<Uri> uris)
+uri_data_provider::uri_data_provider(std::vector<Uri> uris, bool ignore_missing_paths)
```

Comment:
what does it mean to ignore missing paths? Does it mean if a path does not exist but the user said it did we will pretends as if it had? This doesnt make much sense to me.

Reply:
See my comment above